reference/profiles.rst

```diff
@@ -217,6 +244,64 @@ The ``include()`` statement has to be at the top of the profile file:
     zlib:CC=/usr/bin/clang
     zlib:CXX=/usr/bin/clang++
 
+
+.. _profiles_configuration_data_operators:
```

Comment:
Why this section is not right after Configuration **data types** section and has **profile compositon** and **profile includes** in the middle of both?